modules/custom/az_publication/config/install/az_publication.az_citation_style.az_bluebook.yml

```diff
@@ -3,5 +3,5 @@ status: true
 font-style=\"italic\"/>\r\n        </else-if>\r\n        <else-if position=\"subsequent\">\r\n          <group delimiter=\", \">\r\n            <group delimiter=\" \">\r\n              <choose>\r\n                <if type=\"book\" match=\"any\">\r\n                  <text variable=\"volume\"/>\r\n                </if>\r\n              </choose>\r\n              <text macro=\"author-short\"/>\r\n            </group>\r\n            <group delimiter=\" \">\r\n              <text value=\"supra\" font-style=\"italic\"/>\r\n              <text value=\"note\"/>\r\n              <text variable=\"first-reference-note-number\"/>\r\n              <text macro=\"at_page\"/>\r\n            </group>\r\n          </group>\r\n        </else-if>\r\n        <else>\r\n          <group delimiter=\", \">\r\n            <group delimiter=\" \">\r\n              <choose>\r\n                <if type=\"book\" match=\"any\">\r\n                  <text variable=\"volume\"/>\r\n                </if>\r\n              </choose>\r\n              <text macro=\"author\"/>\r\n            </group>\r\n            <text macro=\"source\"/>\r\n            <text macro=\"access\"/>\r\n          </group>\r\n        </else>\r\n      </choose>\r\n    </layout>\r\n  </citation>\r\n  <bibliography et-al-min=\"4\" et-al-use-first=\"1\">\r\n    <layout suffix=\".\" delimiter=\"; \">\r\n      <choose>\r\n        <if position=\"ibid-with-locator\">\r\n          <group delimiter=\" \">\r\n            <text term=\"ibid\" text-case=\"capitalize-first\" font-style=\"italic\"/>\r\n            <text macro=\"at_page\"/>\r\n          </group>\r\n        </if>\r\n        <else-if position=\"ibid\">\r\n          <text term=\"ibid\" text-case=\"capitalize-first\" font-style=\"italic\"/>\r\n        </else-if>\r\n        <else-if position=\"subsequent\">\r\n          <group delimiter=\", \">\r\n            <group delimiter=\" \">\r\n              <choose>\r\n                <if type=\"book\" match=\"any\">\r\n                  <text variable=\"volume\"/>\r\n                </if>\r\n              </choose>\r\n              <text macro=\"author-short\"/>\r\n            </group>\r\n            <group delimiter=\" \">\r\n              <text value=\"supra\" font-style=\"italic\"/>\r\n              <text value=\"note\"/>\r\n              <text variable=\"first-reference-note-number\"/>\r\n              <text macro=\"at_page\"/>\r\n            </group>\r\n          </group>\r\n        </else-if>\r\n        <else>\r\n          <group delimiter=\", \">\r\n            <group delimiter=\" \">\r\n              <choose>\r\n                <if type=\"book\" match=\"any\">\r\n                  <text variable=\"volume\"/>\r\n                </if>\r\n              </choose>\r\n              <text macro=\"author\"/>\r\n            </group>\r\n            <text macro=\"source\"/>\r\n            <text macro=\"access\"/>\r\n          </group>\r\n        </else>\r\n      </choose>\r\n    </layout>\r\n  </bibliography>\r\n</style>"
```

Comment:
This adds `<date variable=\"issued\" prefix=\" \">\r\n`

Reply:
https://github.com/citation-style-language/styles/issues/6805

Reply:
https://github.com/citation-style-language/styles/pull/6807